### ISSUE TITLE
Can't just make checkboxes and let'em unchecked

### ISSUE THREAD

**Issue by brunovcosta** [taromero/swal-forms#7]
just try

```
swal.withForm({
      title: 'More complex Swal-Forms example',
      text: 'This has different types of inputs',
      showCancelButton: true,
      confirmButtonColor: '#DD6B55',
      confirmButtonText: 'Get form data!',
      closeOnConfirm: true,
      formFields: [
          { name: 'skills', value: 'JS', type: 'checkbox' },
          { name: 'skills', value: 'Ruby', type: 'checkbox' },
          { name: 'skills', value: 'Java', type: 'checkbox' }
      ]
  }
```

Now, submit with all checkboxes unmarked. we get an Error: `Uncaught TypeError: Reduce of empty array with no initial value`.

**Comment by brunovcosta** [comment]
solved in 6e9fc62ddef7d55332a1e1c1139d6adf8f4c29a2

**Comment by taromero** [comment]
I've tried to reproduce this but couldn't. It might be related with testing it with different versions of the sweet-alert library. Which version did you used?

Could you provide a reproduction case on JsFiddle or something similar? Thanks!

**Comment by brunovcosta** [comment]
I used the same version of sweet-alert that has found in the live-demo directory.  I used chromium `Version 47.0.2526.106 Ubuntu 14.04 (64-bit)`

Here is the [reproduction](https://jsfiddle.net/rqgjjwrz/). Try uncheck all fields and click in "get form data".

**Comment by taromero** [comment]
Thanks for the reproduction case, I could reproduce it with that. Please add a separate PR with that specific fix so I can merge it independently of other changes introduced on [this PR](https://github.com/taromero/swal-forms/pull/6).

**Comment by brunovcosta** [comment]
I'm not too familiar with git, and still didn't find an way to select commits in PR. I just added tab navigation too.

**Comment by taromero** [comment]
Fixed on #9.
